.github/scripts/generate_binary_build_matrix.py

```diff
@@ -128,9 +125,6 @@ def generate_libtorch_matrix(os: str, abi_version: str,
         if os == "linux":
             arches += CUDA_ARCHES
             arches += ROCM_ARCHES
-        elif os == "windows":
-            # We don't build CUDA 10.2 for window see https://github.com/pytorch/pytorch/issues/65648
-            arches += list_without(CUDA_ARCHES, ["10.2"])
```

Comment:
this is list_without 10.2 for windows. If removing this, windows would be skipped for cuda?

Reply:
You totally right. Added appending arches for windows. Thanks you very much!